### PR TITLE
Use EUI64 from NVS

### LINKED ISSUE #3: Use the MAC address from NVS
Chip ID is 24AA025E64

Currently we set MAC address either hard coded fixed or randomly set.
We need to use the value provided in the onboard chip.

### PR DESCRIPTION
**Pull request by mtusnio**
This closes #3

### CODE REVIEW

File: platform/mikro-e/net-init.c [hunk 1]
```diff
@@ -48,6 +50,44 @@
 #define PRINTF(...)
 #endif
 
+#define NVS_I2C_FREQUENCY   (300000)
+#define NVS_WRITE_ADDRESS   (0xA0)
+#define NVS_READ_ADDRESS    (0xA1)
+#define NVS_EUI_ADDRESS     (0xF8)
+
+static uint64_t
+get_mac_address()
```
Comment (francois-berder-imgtec) [inline platform/mikro-e/net-init.c]: void argument

File: platform/mikro-e/net-init.c [hunk 2]
```diff
@@ -48,6 +50,44 @@
 #define PRINTF(...)
 #endif
 
+#define NVS_I2C_FREQUENCY   (300000)
```
Comment (francois-berder-imgtec) [inline platform/mikro-e/net-init.c]: 10kohm pull up on SCL and SDA. According to datasheet, this is the typical value of pull-up resistor when speed is 100kHz.

**Comment by mtusnio** [comment]
Review comments fixed

File: platform/mikro-e/net-init.c [hunk 3]
```diff
@@ -72,7 +112,7 @@ net_init()
   #ifdef __USE_CC2520__
     cc2520_get_random((uint8_t *)&longaddr, sizeof(longaddr));
   #elif  __USE_CA8210__
-    ca8210_get_random((uint8_t *)&longaddr, sizeof(longaddr));
+    longaddr = get_mac_address();
```
Comment (francois-berder-imgtec) [inline platform/mikro-e/net-init.c]: Move it to another commit.

Comment (abhijit-mahajani-imgtec) [inline platform/mikro-e/net-init.c]: Is there any chip specific functionality in get_mac_address?

Comment (mtusnio) [inline platform/mikro-e/net-init.c]: The TI chips schematics show that they don't have the NVS module, so to maintain backwards compatibility we have to use the random function for cc2520

Comment (abhijit-mahajani-imgtec) [inline platform/mikro-e/net-init.c]: Ok
